sormas-ui/src/main/java/de/symeda/sormas/ui/task/TaskEditForm.java

```diff
@@ -166,9 +166,6 @@ protected void addFields() {
 				users = FacadeProvider.getUserFacade().getUserRefsByDistrict(district, true);
 			} else if (region != null) {
 				users = FacadeProvider.getUserFacade().getUsersByRegionAndRoles(region);
-			} else {
```

Comment:
I think we still need this code because having an empty user list would be a problem. With pseudonymized user names it would be weird, but functional.